websites/G/Gartic.io/presence.ts

```diff
@@ -1,38 +1,41 @@
-const presence = new Presence({
-		clientId: "808668919635247104",
-	}),
-	browsingTimestamp = Math.floor(Date.now() / 1000);
+/* eslint-disable one-var */
```

Comment:
Fix the issue rather than ignoring eslint
```suggestion
```

Reply:
Ok, I just thought this rule resulted into less readable code. Will fix it in a new commit :+1:

Reply:
Resolved in https://github.com/PreMiD/Presences/pull/7158/commits/8c42b9f28faf672356e3dbc401e30f6529b0bc3a